src/core/agent/services/signifyNotificationService.ts

```diff
@@ -200,10 +210,10 @@ class SignifyNotificationService extends AgentService {
       const exchange = await this.props.signifyClient
         .exchanges()
         .get(notif.a.d);
-      const existingCredential = await this.props.signifyClient
-        .credentials()
-        .get(exchange.exn.e.acdc.d)
-        .catch(() => undefined);
+      const existingCredential =
+        await this.credentialStorage.getCredentialMetadata(
```

Comment:
This returns null if it's not found so we should do the same as the `if (!ourIdentifier)` block below and mark the notification. For simplicity we can just do it together in the same if block

Reply:
if (!ourIdentifier || !existingCredential) {
        await this.markNotification(notif.i);
        return;
      }
      
 If we do this, it will skip the issue of a new credential case

Reply:
Yeah sorry, of course, I need more coffee :D nevermind :P